src/widget/OktaSignIn.js

```diff
@@ -167,6 +167,14 @@ var OktaSignIn = (function () {
     OktaAuth = require('@okta/okta-auth-js/jquery');
     Util = require('util/Util');
     LoginRouter = require('LoginRouter');
+    Logger = require('util/Logger');
+
+    Logger.warn(
```

Comment:
Can this be move into `util/logger.js` The actual logging concerns ideally shouldn't leak into this class?

Reply:
I'm interpreting "logging concerns" a couple of different ways, so hopefully I address your comment below.

We want to output this log statement **only** when the SIW is instantiated. Therefore, adding this message to the `Logger` class constructor would produce this error message whenever it is imported and used in another file.

Or - are you saying pull out the message into a separate method: For example - `Logger.initialWarning()`. I do not have a preference with this one, but this design does seem like an anti-pattern. I'll defer to @robertjd for a decision here.

In the future, this string will be localized and reduced to something similar to:
```javascript
Logger.warn(loc('console.developer.inital.warning'));
```
^ Where the `loc()` param is a key lookup.

Reply:
it seems like an implementation detail of the logger class (not the sign in widget). When the logger object is created this message could be recorded. (meaning you wouldn't need a `initialWarning` function)

Reply:
Gotcha. Problem with this approach is whenever this class is imported, we'd log that error.  Take the following example:

```
1. Initialize the widget (Log #1)
2. Pass in unsupported parameter (Log #2)
3. Resort the the "default" success/error handler (Log #3)
...
```

```javascript
// Widget instantiation code

// Logs #1 and #2
var signIn = new OktaSignIn({
  baseUrl: 'https://foo.com',
  fakeKey: 'bar'
});

// Logs #3
signIn.renderEl({el: '#domElement'});
```
Each scenario numbered above is handled in separate files, requiring us to import in the `Logger` class.

Reply:
@rchild-okta - In practice, it make sense to include it in `Logger.js` and output the message once that file loads. A problem arises when we're attempting to validate that `Logger.warn` was called via our jasmine tests.

After some work, I wasn't able to capture this call via a `spyOn` method, as `Logger.warn` origionally gets called from `Expect.js`.

Reply:
Per offline convo, we're going to keep this `Logger` statement in its current location